src/shared/components/chevron/Chevron.tsx

```diff
@@ -38,19 +37,16 @@ const Chevron = (props: Props) => {
     { [styles.chevron_animated]: props.animate }
   );
 
-  const Wrapper = props.onClick ? 'button' : 'span';
-
   const wrapperClasses = classNames(styles.wrapper, props.className);
 
   const wrapperProps = {
-    onClick: props.onClick,
     className: wrapperClasses
   };
 
   return (
-    <Wrapper {...wrapperProps}>
+    <span {...wrapperProps}>
```

Comment:
Do we even need this span?

Reply:
It seems to affect the styling in some places if I remove it. Should we leave the cleanup of this component to a separate PR?

Reply:
Ok, but we shouldn't forget 😄

Reply:
Created a JIRA for this: https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1722